### DISCUSSION TITLE
Importing large list of user emails to auth.users without email confirmation (verification)

### DISCUSSION THREAD

**Discussion by tristankirkpatrick** [supabase/supabase#18336] [Questions]
Hi,

I can successfully create users using the web UI, ticking the 'Auto Confirm User' checkbox. I'd like to be able to dump a CSV into the auth.users table directly, so they can use magic link verification instantly, but I get the following error. 

Is there a workaround for this please?
`SQL Error [428C9]: ERROR: cannot insert a non-DEFAULT value into column "confirmed_at"
  Detail: Column "confirmed_at" is a generated column.
  ERROR: cannot insert a non-DEFAULT value into column "confirmed_at"
  Detail: Column "confirmed_at" is a generated column.
  ERROR: cannot insert a non-DEFAULT value into column "confirmed_at"
  Detail: Column "confirmed_at" is a generated column.
`

**Comment by nestorbfgo** [comment]
@tristankirkpatrick Hi, can you do it?
